tests/Unit/TecanScanner/TecanScannerTest.php

```diff
@@ -14,31 +14,40 @@ final class TecanScannerTest extends TestCase
     public function testCreateFromStringEmpty(): void
     {
         $this->expectExceptionObject(new TecanScanEmptyException());
-        TecanScanner::parseRawContent('');
+        $rawContent = '';
+        TecanScanner::parseRawContent($rawContent);
+        self::assertFalse(TecanScanner::isValidRawContent($rawContent));
     }
 
     public function testCreateFromUnexpectedLineCount(): void
     {
         $this->expectExceptionObject(new WrongNumberOfWells(96, 2));
-        TecanScanner::parseRawContent("rackid,SA00411242\nA1,FD13945423\nB1,FD32807353");
+        $rawContent = "rackid,SA00411242\nA1,FD13945423\nB1,FD32807353";
+        self::assertFalse(TecanScanner::isValidRawContent($rawContent));
```

Comment:
minimize the code inbetween setting up the expectation for an exception and triggering the exception